src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -465,6 +630,7 @@ private boolean runIntegrationTestOnZipFile(
             // There should be no schema records matching the deleted namespace.
             assertThat(schemaCount, is(0));
         } catch (SQLException | InvalidNamespaceException e) {
+            LOG.error("An error occurred while deleting a schema!");
             e.printStackTrace();
```

Comment:
Can you add the exception as the second arg in the error log (and remove `printStackTrace`)? Doing this causes the stack trace to be included in the error log and helps with formatting/keeping the exception close to the error log.